core-web/libs/portlets/dot-ema/src/lib/feature/dot-ema.component.html

```diff
@@ -1,36 +1,50 @@
-<div class="flex gap-3 toolbar align-items-center">
-    <h4>{{ title$ | async }}</h4>
+<ng-container *ngIf="vm$ | async as vm">
+    <div class="flex gap-3 toolbar align-items-center">
+        <h4>{{ vm.pageTitle }}</h4>
 
-    <select
-        id="language_id"
-        [ngModel]="language_id$ | async"
-        (change)="onChange($event)"
-        data-testId="language_id"
-        aria-label="language_id"
-        name="language_id">
-        <option *ngFor="let language of languages" value="{{ language.value }}">
-            {{ language.name }}
-        </option>
-    </select>
-
-    <!-- This is for demo purpose, going away -->
-    <select
-        id="url"
-        [ngModel]="url$ | async"
-        (change)="onChange($event)"
-        aria-label="url"
-        name="url">
-        <option *ngFor="let page of pages" value="{{ page.value }}">
-            {{ page.name }}
-        </option>
-    </select>
-</div>
-<div class="iframe-wrapper">
-    <iframe
-        [src]="iframeUrl$ | async | safeUrl"
-        data-testId="iframe"
-        frameborder="0"
-        width="100%"
-        height="100%"
-        title="Editor Page"></iframe>
-</div>
+        <select
+            id="language_id"
+            [ngModel]="vm.language_id"
+            (change)="onChange($event)"
+            data-testId="language_id"
+            aria-label="language_id"
+            name="language_id">
+            <option *ngFor="let language of languages" value="{{ language.value }}">
+                {{ language.name }}
+            </option>
+        </select>
+    </div>
+    <div class="iframe-wrapper">
+        <iframe
+            #iframe
+            [src]="vm.iframeUrl | safeUrl"
+            [title]="host"
+            data-testId="iframe"
+            style="border: none"
+            width="100%"
+            height="100%"
+            title="Editor Page"></iframe>
+    </div>
+    <p-dialog
+        #dialog
+        [visible]="vm.dialogVisible"
+        [style]="{ height: '90vh', width: '90vw' }"
+        [header]="vm.dialogHeader"
+        [draggable]="false"
```

Comment:
We can create a directive for pre-configure and set by default `[draggable]="false" [resizable]="false"
        [maximizable]="true"
        [modal]="true"` ?I have the same combination in something I'm working on.

Reply:
Maybe...

Reply:
Maybe is not worth it right now.